app/org/views.py

```diff
@@ -1,33 +1,62 @@
-from flask import render_template
-from ..models import EditableHTML
-from flask.ext.login import (
-    login_required,
-    login_user,
-    logout_user,
-    current_user
-)
-from . import main
-
+from flask import render_template, flash
+from flask_login import (login_required, current_user)
+from . import org
+from ..decorators import organization_required
+from .forms import OrganizationForm
+from ..models import Organization
+from .. import db
 
 
 @org.route('/')
```

Comment:
make the welcome page render for main when an org is logged in (now the default is an admin welcome)

Reply:
Right now the main page that's rendered is just the main.index page. I did this primarily bc  that'll eventually be the search page. Organization dashboard is what's being used as the "main" page for an organization.

Reply:
okay. Eventually we have to clarify between the Admin-panel and the organization panel because currently logging in as an admin is the same as logging in as an organization (i can access the organization site on login and the admin when i log in as flask-base-admin). We'll define a new task to parse out the difference